worlds/factorio/docs/en_Factorio.md

```diff
@@ -24,11 +24,21 @@ place them directly into the player's inventory.
 ## What does another world's item look like in Factorio?
 
 In Factorio, items which need to be sent to other worlds appear in the tech tree as new research items. They are
-represented by the Archipelago icon, and must be researched as if it were a normal technology. Upon successful
+represented by the Archipelago icon, and must be researched as if they were normal technologies. Upon successful
 completion of research, the item will be sent to its home world.
 
 ## When the engineer receives an item, what happens?
 
 When the player receives a technology, it is instantly learned and able to be crafted. A message will appear in the chat
 log to notify the player, and if free samples are enabled the player may also receive some items directly to their
 inventory.
+
+## What is EnergyLink?
+
+EnergyLink is an energy storage shared by all worlds in a multiworld. In Factorio, if enabled in the player settings,
+EnergyLink Bridge buildings can be crafted and placed, which allow depositing excess energy and supplementing energy
+deficits, much like Accumulators.
```

Comment:
maybe make it clear that it has different effect on different games, as to my knowledge The Witness has it. and the alpha Satisfactroy Mod for Archipelago also has it

Reply:
How does The Witness interface with EnergyLink? I can't find anything about it.

I'm already implying that different games may use EnergyLink differently with the "EnergyLink is … // In Factorio, …" wording. We could elaborate, but it seems unnecessary here, especially as there are no games doing anything different with it yet. (Even Satisfactory would use it in exactly the same way, no?)

Reply:
Oke the way it currently reads to me is that that its an energy storage in Factorio
For Satisfactory it will work mostly the same yes
The witness had more daring plans but maybe @NewSoupVi can explain that

Reply:
I can see that reading now and I've pushed a clarification.

I don't think we should explain here how each game deals with EnergyLink or even provide a list of games, as this should be Factorio-specific. Something like a FAQ entry is probably a better idea once there is more supported games.